### ISSUE TITLE
Opinions on including DSMZ's PNU & BacDive?

### ISSUE THREAD

**Issue by katrinleinweber** [ropensci/taxize#683]
The German Collection of Microorganisms and Cell Cultures (DSMZ) has a web service called "[Prokaryotic Nomenclature up-to-date](https://www.dsmz.de/bacterial-diversity/prokaryotic-nomenclature-up-to-date.html)":

> PNU is a compilation of all names of Bacteria and Archaea which have been validly published according to the Bacteriological Code since 1. Jan. 1980, and nomenclatural changes which have been validly published since. It will be updated with the publication of each new issue of […] [IJSEM](https://ijs.microbiologyresearch.org/content/journal/ijsem).

Would that we be a relevant data source to add?

PS: Related to [BacDiveR/#34](https://github.com/TIBHannover/BacDiveR/issues/34) because that's about another DSMZ web service. Same questions there: relevant for `taxize`?

**Comment by sckott** [comment]
thanks for the issue @katrinleinweber 

I'm not familiar with DSMZ but does look interesting. I'll have a look. @zachary-foster are you familiar with DSMZ at all? any thoughts?

**Comment by sckott** [comment]
hi @katrinleinweber i've registered and waiting on approval, but i think at first glance it's probably worth incorporating here.

**Comment by zachary-foster** [comment]
Hi @sckott, sorry didnt see this until now. Nope, I dont know anything about it unfortunately.
